src/IW/App/Error.hs

```diff
@@ -118,15 +123,22 @@ data AppErrorType
     | UrlDownloadFailed Url
     deriving (Show, Eq)
 
--- | Map the @github@ library's @Error@ type into AppErrorType.
+newtype CabalPError = CabalPError { unCabalPError :: PError }
```

Comment:
I think that `CabalParseError` is not that long, but it brings much more into the context immediately. Anyway, the documentation would be handy here.

Also, by the style guide the indentation should be modified a bit:
```suggestion
newtype CabalPError = CabalPError 
    { unCabalPError :: PError 
    } 
```

Reply:
@vrom911 I was ignorant of how haddock works and replaced the `'...'` with `@...@` for no good reason. I'm going to change it back. I thought the `'...'` were for something else.

Reply:
Oh, no worries! Here you can check what `''` means in haddock:
* https://haskell-haddock.readthedocs.io/en/latest/markup.html#hyperlinked-identifiers

Reply:
@vrom911 I'm sorry I didn't quite understand the first sentence. Should I choose a different name for `CabalParseError`? I wrote some docs for it.

Reply:
@rashadg1030 I think @vrom911 means that `CabalPError` should be renamed to `CabalParseError` and I agree with that. It's more readable. You will have collisions with the `AppErrorType` constructor but you can rename the constructor to just `CabalError` to avoid clashes.